### PR TITLE
Form login

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

File: frontend/src/components/FormCreateCause.jsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    img: null,
+    resume: "",
+    website: "",
+    id_tag: []
+  });
+
+  const handleSubmit = () => {
+    const data = asso;
+
+    console.log(data);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateCause.jsx]: Console log

File: frontend/src/components/FormCreateCause.jsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    axios
+      .post(`${apiCall}/association`, data)
+      .then(res => {
+        console.log(`Booyah! ${res.data}`);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateCause.jsx]: console log

File: frontend/src/components/FormCreateCause.jsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+      })
+      .catch(err => {
+        console.log(`Nay! ${err}`);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateCause.jsx]: An alerte should be better

**Review by TheoBoschetti — CHANGES_REQUESTED** [review]

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by BastienSaulnier — DISMISSED** [review]
SnE

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**